### PR TITLE
fix: Member list is randomly sorted 

### PR DESCRIPTION
**Pull request by Keanby**

### CODE REVIEW

**Review by debdutdeb — CHANGES_REQUESTED** [review]
Should return a sorted list from the be already, configurable by some sort param.

Please check many of our other endpoint code that accepts a sort parameter which then returns a sorted list based on the key to the client.

File: apps/meteor/client/views/room/contextualBar/RoomMembers/RoomMembersWithData.tsx [hunk 1]
```diff
@@ -95,6 +95,20 @@ const RoomMembersWithData = ({ rid }: { rid: IRoom['_id'] }): ReactElement => {
 	if (state.tab === ROOM_MEMBERS_TABS.ADD) {
 		return <AddUsers rid={rid} onClickBack={handleBack} reload={reload} />;
 	}
```
Comment (KevLehman) [inline apps/meteor/client/views/room/contextualBar/RoomMembers/RoomMembersWithData.tsx:97]: Shouldn't this sort be made on server?

**Review by KevLehman — CHANGES_REQUESTED** [review]
See comments

**Comment by KevLehman** [comment]
Thanks! I'll close this one in favor of #29228 :)

**Pull request closed without merging**